circle.yml

```diff
@@ -15,7 +15,7 @@ test:
         parallel: true
     - python setup.py install:
         parallel: true
-    - pip install coala-bears --pre -U:
+    - pip install coala-bears colorama==0.3.6 --pre -U:
```

Comment:
does it actually work? like arent ``coala-bears`` and ``colorama`` 2 different packages?